src/pudl_rmi/__init__.py

```diff
@@ -1,12 +1,44 @@
 """Connecting depreciation data, FERC1 and EIA for RMI."""
 
 import pkg_resources
+from pathlib import Path
 
-import pudl_rmi.connect_deprish_to_eia
-import pudl_rmi.connect_deprish_to_ferc1
-import pudl_rmi.connect_ferc1_to_eia
-import pudl_rmi.deprish
 import pudl_rmi.make_plant_parts_eia
+import pudl_rmi.deprish
+import pudl_rmi.coordinate
+import pudl_rmi.connect_ferc1_to_eia
+import pudl_rmi.connect_deprish_to_ferc1
+import pudl_rmi.connect_deprish_to_eia
+
+REPO_DIR = Path(__file__).resolve().parent.parent.parent
+INPUTS_DIR = REPO_DIR / 'inputs'
+"""
+Directory of input files that are used in generating the RMI outputs.
+
+These files are required inputs and should be checked into the repo. Some of
+these files (even the excel files) are updated during some of the processing.
+If these files don't exist, then many of the outputs will fail.
+"""
+OUTPUTS_DIR = REPO_DIR / 'outputs'
+"""
+Directory of output files that are generated from the RMI processes.
+
+Nothing needs to be checked into this directory for any of the RMI processes
+to run. If using ``pudl_rmi.coordinate.Output()`` these files will be generated
+and stored as pickled dataframes. If these files do exist,
+``pudl_rmi.coordinate.Output()`` will either grab them or clobber them.
+"""
+
+FILE_PATH_TRAIN_FERC_EIA = INPUTS_DIR / 'train_ferc1_eia.csv'
```

Comment:
Great, these are much more consistent!  Note here you've got `FERC` in the variable and `ferc1` in the filename.

If you wanted to, I think you could also do away with the `FILE_PATH` prefix and instead use type hints and a file type suffix to get across more information (that it's a file and also what kind of file) in less space, and enable better tab completion / documentation in your IDE / Jupyter.  E.g.
```py
TRAIN_FERC1_EIA_CSV: Path = INPUTS_DIR / 'train_ferc1_eia.csv'
DEPRISH_COMMON_LABELS_XLSX: Path = INPUTS_DIR / 'deprish_common_labels.xlsx'
PLANT_PARTS_EIA_PKL: Path = OUTPUTS_DIR / 'plant_parts_eia.pkl.gz'
```
since the methods for reading / writing the paths will differ based on file type.

Reply:
good idea!